.github/workflows/tav.yml

```diff
@@ -24,21 +24,34 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
     outputs:
-      versions: ${{ steps.generate-matrix.outputs.versions }}
-      modules: ${{ steps.generate-matrix.outputs.modules }}
+      versions: ${{ steps.transform.outputs.versions }}
+      modules: ${{ steps.transform.outputs.modules }}
     steps:
-    - uses: actions/checkout@v3
-    ## TODO: use elastic/apm-pipeline-library/.github/actions/version-framework
-    ##       as soon as https://github.com/elastic/apm-pipeline-library/issues/2171 is done.
-    - id: generate-matrix
-      run: .ci/generate-github-tav-matrix.sh
+
+      - uses: actions/checkout@v3
+
+      - id: transform
+        name: Transform comment to the supported matrix
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require('fs')
+            let matrix
+            try {
+              matrix = JSON.parse(fs.readFileSync('./.ci/tav.json'))
+            } catch (err) {
+              core.setFailed(`Error loading './.ci/tav.json': ${err}`)
+              return
+            }
+            core.setOutput('modules', matrix.modules)
+            core.setOutput('versions', matrix.versions)
 
   test-tav:
     needs: prepare-matrix
     runs-on: ubuntu-latest
     timeout-minutes: 40
     strategy:
-      max-parallel: 30
+      max-parallel: 15
```

Comment:
There is no need to run in parallel more since it's a merge build, so it's not happening on PRs.

Reply:
I don't understand what you mean here. Our test matrix is approximately 250 jobs now. If we run only 15 in parallel it may take twice as long to complete them all. Currently (see some examples at https://github.com/elastic/apm-agent-nodejs/actions/workflows/tav.yml) a TAV run is taking 1.5h to complete. It would be unfortunate if it takes 3h to complete.

Is there a reason to want lower max-parallel? Is there added cost?